### PR TITLE
Page tracking endpoint added

### PR DESCRIPTION
**Pull request by helgeblod**
* All integrations now respond to the page-endpoint

Didn't update the README as the track-endpoint is not documentet there either. Might be a good idea to document the api as the api grows, but maybe it should be done using a tool that is more apropriate (eg http://raml.org/ or similar)

### CODE REVIEW

File: handlers/fake_integration.go [hunk 1]
```diff
@@ -17,6 +17,11 @@ func (fi FakeIntegration) Track(event integrations.Event) error {
 }
 
 // Enabled returns wether or not the integration is enabled/configured
+func (fi FakeIntegration) Page(page integrations.Page) error {
```
Comment (jipiboily) [inline handlers/fake_integration.go]: Comment for that one is wrong :)

File: integrations/integration.go [hunk 2]
```diff
@@ -10,6 +10,8 @@ type Integration interface {
 	// Track forwards the event to the integration
 	Track(event Event) error
 
+	Page(page Page) error
```
Comment (jipiboily) [inline integrations/integration.go]: A comment is missing for that one.

File: integrations/integration.go [hunk 3]
```diff
@@ -74,3 +76,43 @@ func (e Event) Validate() (missingParameters []string) {
 	}
 	return
 }
+
+type Page struct {
```
Comment (jipiboily) [inline integrations/integration.go]: Comment missing here too.

File: integrations/integration.go [hunk 4]
```diff
@@ -44,6 +45,7 @@ func (i Identification) Validate() (missingParameters []string) {
 	return
 }
 
+// Event defines the structure for the incoming event data from the API
 type Event struct {
```
Comment (helgeblod) [inline integrations/integration.go]: Added this as well as it was missing :)

File: integrations/integration.go [hunk 5]
```diff
@@ -44,6 +45,7 @@ func (i Identification) Validate() (missingParameters []string) {
 	return
 }
 
+// Event defines the structure for the incoming event data from the API
```
Comment (helgeblod) [inline integrations/integration.go]: Added this as well as it was missing :)

**Comment by helgeblod** [comment]
Should be fixed now. Merge when you are ready :)

**Comment by jipiboily** [comment]
@helgeblod there you go!

**Pull request merged**